Distribution missing UI index template files

```shellsession
$ unzip -l bottle_openapi_3-0.1.0-py2.py3-none-any.whl
Archive:  bottle_openapi_3-0.1.0-py2.py3-none-any.whl
  Length      Date    Time    Name
---------  ---------- -----   ----
    12104  12-13-2020 19:53   bottle_openapi_3/__init__.py
     1069  01-02-2021 01:05   bottle_openapi_3-0.1.0.dist-info/LICENSE
     1202  01-02-2021 01:05   bottle_openapi_3-0.1.0.dist-info/METADATA
      110  01-02-2021 01:05   bottle_openapi_3-0.1.0.dist-info/WHEEL
       17  01-02-2021 01:05   bottle_openapi_3-0.1.0.dist-info/top_level.txt
      518  01-02-2021 01:05   bottle_openapi_3-0.1.0.dist-info/RECORD
---------                     -------
    15020                     6 files
```

stack trace
```
/usr/local/lib/python3.9/site-packages/bottle_openapi_3/__init__.py:29: in <module>
    with open(SWAGGER_UI_INDEX_TEMPLATE_PATH, 'r') as f:
E   FileNotFoundError: [Errno 2] No such file or directory: '/usr/local/lib/python3.9/site-packages/bottle_openapi_3/vendor/swagger-ui-3.38.0-dist/index.html.st'
```

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -2,7 +2,7 @@ include LICENSE
 include MANIFEST.in
 include README.rst
 include requirements.txt
-include bottle_openapi_3/__init__.py
+recursive-include bottle_openapi_3 *.py *.png *.html *.html.st *.css *.js
 
 recursive-exclude * __pycache__
 recursive-exclude * *.py[co]

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,0 +1,3 @@
+[build-system]
+requires = ['setuptools', 'wheel']
+build-backend = 'setuptools.build_meta'

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,2 +1,41 @@
 [metadata]
-description-file = README.rst
+name = bottle-openapi-3
+url = https://github.com/cope-systems/bottle-openapi-3
+description = OpenAPI Integration for Bottle
+long_description = file: README.rst
+author = Robert Cope
+author_email = robert@copesystems.com
+license = MIT
+platforms = any
+classifiers =
+    Development Status :: 3 - Alpha
+    Environment :: Web Environment
+    Environment :: Plugins
+    Framework :: Bottle
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Internet :: WWW/HTTP :: Dynamic Content
+    Topic :: Software Development :: Libraries :: Python Modules
+
+[options]
+include_package_data = True
+packages = find:
+install_requires =
+    bottle>=0.12.0
+    openapi-core~=0.13.4
+    openapi-spec-validator~=0.2.9
+    six
+tests_require =
+    tox
+    webtest
+
+[options.packages.find]
+include =
+    bottle_openapi_3

--- a/setup.py
+++ b/setup.py
@@ -2,48 +2,9 @@
 import os
 from setuptools import setup
 
-
-def _read(fname):
-    try:
-        return open(os.path.join(os.path.dirname(__file__), fname)).read()
-    except IOError:
-        return ''
-
-
-REQUIREMENTS = [l for l in _read('requirements.txt').split('\n') if l and not l.startswith('#')]
-VERSION = '0.1.0'
+VERSION = '0.1.1'
 
 setup(
-        name='bottle-openapi-3',
         version=VERSION,
-        url='https://github.com/cope-systems/bottle-openapi-3',
         download_url='https://github.com/cope-systems/bottle-openapi-3/archive/v{0}.tar.gz'.format(VERSION),
-        description='OpenAPI Integration for Bottle',
-        long_description=_read("README.rst"),
-        author='Robert Cope',
-        author_email='robert@copesystems.com',
-        license='MIT',
-        platforms='any',
-        packages=["bottle_openapi_3"],
-        package_data={"bottle_openapi_3": ["*.png", "*.html", "*.html.st", "*.css", "*.js"]},
-        install_requires=REQUIREMENTS,
-        tests_require=REQUIREMENTS + ["tox", "webtest"],
-        classifiers=[
-            'Development Status :: 3 - Alpha',
-            'Environment :: Web Environment',
-            'Environment :: Plugins',
-            'Framework :: Bottle',
-            'Intended Audience :: Developers',
-            'License :: OSI Approved :: MIT License',
-            'Operating System :: OS Independent',
-            'Programming Language :: Python',
-            'Programming Language :: Python :: 3.5',
-            'Programming Language :: Python :: 3.6',
-            'Programming Language :: Python :: 3.7',
-            'Programming Language :: Python :: 3.8',
-            'Programming Language :: Python :: 3.9',
-            'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
-            'Topic :: Software Development :: Libraries :: Python Modules'
-        ],
-        include_package_data=True
 )

